text/0000-temp-components-lib.md

```diff
@@ -0,0 +1,610 @@
+
+Another alternative that was considered is using an already familiar 3rd party
+under the hood (i.e. Chakra, MUI, etc.). Ultimately, this doesn't meet our need
+and creates more issues regarding everything from maintaining code to managing
+security issues which naturally stem from relying on a 3rd party.
+
+In considering our original core value of Pando: creating a library that is
+framework agnostic, web-components have been considered. However, this creates a
+contradiction in going all in on components which is what we don't want to do. Likewise,
+it creates overly-complexed code to maintain, just to be more marketable to the
+world (i.e. we have all the component frameworks) when we can do the same thing
+with a completely different design that isn't complex (i.e. Headless-styles).
+
+## Adoption strategy
+
+### If we implement this proposal, how will existing Pando developers adopt it?
+
+Users will need to install the `@pluralsight/react` library and only use it in conjunction
+with the Pando Normalize/Font setup [mentioned in the docs](https://design.pluralsight.com/docs/next/learn/get-started/installation/add-to-website). Users will _not_ need to install any other
+Pando libraries to use the React one because it will use them internally.
+
+### Is this a breaking change?
+
+By nature of introducing a completely new design, yes, it will be a breaking change
+but easier for most users to migrate than using Headless-styles due to having an
+API that is familair.
+
+### Can we write a codemod?
+
+Could be worth exploring for _Classic users_. Classic by nature is not 1:1 to Pando
+(and never will be) and is quite honestly very inconsistent in how it designs
+components. However, it might work for the components that are used in the Pando
+UI Kit.
+
+There would need to be the following features:
+
+1. Correct the import paths in each file
+2. Correct the component usage and it's props in each file - without causing bugs
+3. Delete unused Classic component code that is naturally built into Pando (there is a lot)
+4. Delete unused Classic dependencies (see [migration list](https://design.pluralsight.com/docs/next/learn/learn-pando/migration)) from the users package.json
+  a. Recognize whether a project uses Yarn/NPM and re-install everything automatically
+5. Log warnings/errors for old Classic components that are no longer supported but are still around
+  a. Probably a nice interface like "Pando no longer uses the Breadcrumb component, would you like to keep it? (Y/N)"
+
+:warning: This should definitely be approached with caution! :warning:
+
+If we can make this happen, the ROI on business **would be unlike anything
+Pluralsight as a company has ever seen** regarding migrating to a new brand/tooling.
+
+### Should we coordinate with other projects or libraries within Pando?
+
+Yes. All Pando libraries should share the same version number to make using and
+maintaining each package easier for the developer experience.
```

Comment:
Is this section talking about collaboration between teams, or projects within the same team using different versions of pando libs? This part could use some extra clarity in my opinion.

Reply:
Updated.